indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -15,7 +15,8 @@
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
 
-/* global setupListGenerator:false, setupSearchBox:false, applySearchFilters:false, Palette:false */
+/* global setupListGenerator:false, setupSearchBox:false, applySearchFilters:false, Palette:false,
+          avatarPalette: false */
```

Comment:
It was working for me but it won't be problem anymore since we will be using `Palette.userColors`